src/engagement_db_to_analysis/data_wrangling_functions.py

```diff
@@ -0,0 +1,81 @@
+from core_data_modules.util import TimeUtils
```

Comment:
Can we call this file code_imputation_functions instead? I think all of analysis can arguably be described as a data wrangling operation